src/terminal.py

```diff
@@ -57,8 +57,11 @@ def sorting_links(sub_links):
             if("resource/view.php?id" in url or "folder/view.php?id=" in url):
                 res_urls[x].append(url)
             elif("page/view.php?id" in url):
-                notice_urls[x].append(
-                    [url, sub_links[x][y].contents[1].contents[0]])
+                try:
+                    notice_urls[x].append(
+                        [url, sub_links[x][y].contents[1].contents[0]])
+                except IndexError:
```

Comment:
Don't put it only in a try except block. You need to make sure that all the notices also get printed